crowbar_framework/lib/crowbar/upgrade_timeouts.rb

```diff
@@ -38,6 +38,7 @@ def values
         delete_nova_services: @timeouts_config[:delete_nova_services] || 300,
         wait_until_compute_started: @timeouts_config[:wait_until_compute_started] || 60,
         reload_nova_services: @timeouts_config[:reload_nova_services] || 120,
+        stop_nova_services: @timeouts_config[:stop_nova_services] || 120,
```

Comment:
Add this to `configs/upgrade_timeouts.yml` as well. This file is packaged as an example for the users.

Reply:
Done.